wallet/src/adapters/keybase.rs

```diff
@@ -261,3 +305,35 @@ impl WalletCommAdapter for KeybaseWalletCommAdapter {
 		Ok(())
 	}
 }
+
+/// Notify in keybase on receiving a transaction
+fn notify_on_receive(keybase_notify_ttl: u16, channel: String, tx_uuid: String) {
+	if keybase_notify_ttl > 0 {
+		let my_username = whoami();
+		if let Some(username) = my_username {
+			let split = channel.split(",");
+			let vec: Vec<&str> = split.collect();
+			if vec.len() > 1 {
+				let receiver = username;
+				let sender = if vec[0] == receiver {
+					vec[1]
```

Comment:
Keybase allows for multi person channels i.e `me,alice,bob` or `bob,alice,me`. If that's the case the above will fail or even show the wrong sender. One workaround is just to output the channel name, for example: You might have received some grins from {channel}

Reply:
thanks @mcdallas for your review, not sure I understand,  are you talking about 3 or more party payment?  current grin wallet only support 2-party transaction, so the channel will always be `someone,me`, yes?

and if leaving it as the whole `{channel}`, that will also include `me`, it will make user confused which one is the real payer.

Reply:
👍  updated for the limitation of 2 party.